distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,232 @@
+
+.Search Menu Options
+* *Run*: Trigger this search to begin immediately.
+* *Stop*: Stop this search.
+* *Delete*: Remove this search
+* *Duplicate*: Create a copy of this search as a starting point
+* *Search Archived*: Execute the search, but specifically for archived results.
+
+===== Editing a Search
+
+Existing searches and their settings can be updated by clicking on the search in the *Searches* tab of the workspace.
+
+The editable components of a search are defined on these tabs:
+
+* *Basic*: Define a <<_text_search,Text>>, <<_spatial_search,Spatial>>, <<_temporal_search,Temporal>>, or <<_datatype_search,Datatype>> Search.
+* *Advanced*: Advanced query builder to create more specific searches.
+* *Settings*: Set preferences for *Sorting* and *Federation*.
+* *Notifications*: Define the frequency at which to run this search and receive notification of new results
+* *Status*: View details of recent instances of this search.
+
+====== Editing Search Settings
+
+Under the settings tab for each search, edit the default *Sorting* and *Federation* (which sources are searched).
+
+====== Setting Search Frequency
+
+Edit the *Notifications* tab for a search to set the interval at which to run the query.
+
+====== Viewing Search Status
+
+The *Status* tab on the search preview displays the creation date and the sources successfully searched.
+
+==== Viewing Search Results
+
+Returned search results can be refined further, bookmarked, and/or downloaded from the *Results* tab.
+Results sets are color-coded by source as a visual aid.
+There is no semantic meaning to the colors assigned.
+
+. Select a saved query.
+. Perform any of these action upon the results list of the selected query:
+.. Filter results further.
+.. Re-sort list(default is _title_/_ascending_).
+.. Toggle between *list* and *gallery* view.
+
+==== Bookmarks Tab
+
+Search Results that have been bookmarked can be filtered and sorted.
+
+.Bookmarked Result Options
+* *Add to Bookmarks*.
+** Changes to *Remove from Bookmarks* for bookmarked items.
+* *Hide from Future Searches*.
+* *Expand Metacard View*.
+* *Download*.
+
+=== Result Preview Pane
+
+On the *Result Preview* pane, view a summary of information about the selected resource.
+The *Details* tab provides a list of specific metadata.
+
+<<_viewing_revision_history,History>>:: View revision history of this resource.
+<<_editing_associations_on_a_record,Associations>>:: View or edit the relationship(s) between this resource and others in the catalog.
+<<_viewing_metadata_quality,Quality>>:: View the completeness and accuracy of the metadata for this resource.
+<<_exporting_a_record,Actions>>:: Export the metadata/resource to a specific format.
+<<_archiving_a_record,Archive>>:: Remove the selected record from standard search results.
+<<_overwriting_a_record,Overwrite>>:: Overwrite a record.
+
+==== Editing Records
+
+Records can be edited from the *Summary* or *Details* tabs.
+
+==== Editing Associations on a Record
```

Comment:
There is also a graphical view for the Associations. May want to note this here.